src/components/Attachments/AttachmentCarousel/Pager/AttachmentCarouselPagerContext.ts

```diff
@@ -0,0 +1,16 @@
+import {createContext} from 'react';
+import type PagerView from 'react-native-pager-view';
+import type {SharedValue} from 'react-native-reanimated';
+
+type AttachmentCarouselPagerContextType = {
```

Comment:
```suggestion
type AttachmentCarouselPagerContextValue = {
```

nit, we have been using `...ContextValue` in most of the other contexts.